types/places.js/index.d.ts

```diff
@@ -0,0 +1,340 @@
+         * Default: Search on the whole planet.
+         */
+        countries?: string[];
+
+        /**
+         * Force to first search around a specific latitude longitude.
+         * @remarks
+         * The option value must be provided as a string: `latitude, longitude` like `12.232,23.1`.
+         * The default is to search around the location of the user determined via his IP address (geoip).
+         */
+        aroundLatLng?: string;
+
+        /**
+         * Whether or not to first search around the geolocation of the user found via his IP address.
+         * @defaultValue: `true`
+         */
+        aroundLatLngViaIP?: boolean;
+
+        /**
+         * Radius in meters to search around the latitude longitude.
+         * @remarks
+         * Otherwise a default radius is automatically computed given the area density.
+         */
+        aroundRadius?: number;
+
+        /**
+         * Filters the results inside the area defined by the two extreme points of a rectangle.
+         *
+         * @remarks
+         * {@link https://www.algolia.com/doc/guides/managing-results/refine-results/geolocation/#filter-inside-rectanglepolygonal-area | See guide} or
+         * {@link https://www.algolia.com/doc/api-reference/api-parameters/insideBoundingBox/ | API reference}.
+         * Format: `topRightLat, topRightLng, bottomLeftLat, bottomLeftLng`
+         * @example
+         * "60, 16, 40, -4"
+         */
+        insideBoundingBox?: string;
+
+        /**
+         * Filters the results inside the area defined by a shape.
+         * @remarks
+         * {@link https://www.algolia.com/doc/guides/managing-results/refine-results/geolocation/#filter-inside-rectanglepolygonal-area | See guide} or
+         * {@link https://www.algolia.com/doc/api-reference/api-parameters/insidePolygon/ | API reference}.
+         * Format: `p1Lat, p1Lng, p2Lat, p2Lng, p3Lat, p3Lng...`
+         */
+        insidePolygon?: string;
+
+        /**
+         * Controls whether the {@link https://www.algolia.com/doc/api-reference/api-methods/search/#method-response-_rankinginfo | _rankingInfo} object should be included in the hits.
+         * @remarks
+         * The _rankingInfo object for a Places query is slightly different from a regular Algolia query and you can read up more about the difference and how to leverage them in
+         * {@link https://community.algolia.com/places/examples.html#using-_rankinginfo | our guide}.
+         * @defaultValue `false`
+         */
+        getRankingInfo?: boolean;
+
+        /**
+         * Ask and use the device location.
+         */
+        useDeviceLocation?: boolean;
+
+        /**
+         * Allows you to override computed query parameters sent to the {@link https://community.algolia.com/places/rest.html | Algolia Places REST API}.
+         */
+        computeQueryParams?: algoliasearch.QueryParameters;
+    }
+
+    class PlacesAutocomplete extends EventEmitter {
```

Comment:
```suggestion
    interface PlacesAutocomplete extends EventEmitter {
```

`places` doesn't export the class, so there's no actual *value* name `PlacesAutocomplete`